webpack.common.config.ts

```diff
@@ -788,6 +790,16 @@ module.exports = {
       minify: htmlMinifyConfig,
       inject: false
     }),
+    new HtmlWebpackPlugin({
+      chunks: ['volunteer'],
```

Comment:
Please add an entry for this into the lighthouse configs.

Reply:
Done.